### ISSUE TITLE
text-* css classes don't pass colour contrast

### ISSUE THREAD

**Issue by cfarquharson** [wet-boew/GCWeb#319]
Every colour except the red one, fail color contrast requirements against the background color
![text-color](https://f.cloud.github.com/assets/2319199/1673953/e4ef2088-5cf0-11e3-8d71-0f060b4eee42.PNG)
